Lib/idlelib/idle_test/test_sidebar.py

```diff
@@ -1,4 +1,7 @@
 """Test sidebar, coverage 93%"""
```

Comment:
```suggestion
"""Test sidebar, coverage 84%"""
```

Reply:
I hope to increase coverage by adding more tests. So I won't be adding this change quite yet.

Reply:
Fine.  We can leave this unresolved as a reminder.

Reply:
I've updated this now that I'm done removing dead code and adding tests (now at 96%!)